airflow/operators/bash.py

```diff
@@ -72,6 +72,10 @@ class BashOperator(BaseOperator):
         users in the Web UI. Most of the default template variables are not at
         risk.
 
+        Add a space after the script name when directly calling a Bash script with the
+        ``bash_command`` argument.  This is because Airflow tries to apply a Jinja template to it,
+        which will fail.
```

Comment:
```suggestion
        Add a space after the script name when directly calling a a ``.sh`` script with the
        ``bash_command`` argument -- for example ``bash_command="my_script.sh "``.  This
        is because Airflow tries to apply load this file and process it as a Jinja template to
        it ends with ``.sh``, which will likely not be what most users want.
```